app/controllers/welcome_controller.rb

```diff
@@ -28,6 +29,18 @@ def mentor
     @report = params[:report].blank? ? "basics" : ERB::Util.html_escape(params[:report])
   end
 
+  def participant
```

Comment:
I would add a check here to ensure `@current_user.person_type` is a `Participant`